depthai_demo.py

```diff
@@ -216,6 +237,14 @@ def startLoop(self):
                             cv2.putText(frame, packet.stream_name, (25, 25), cv2.FONT_HERSHEY_SIMPLEX, 1.0, (0, 0, 255))
                             cv2.putText(frame, "fps: " + str(frame_count_prev[window_name]), (25, 50), cv2.FONT_HERSHEY_SIMPLEX, 1.0, (0, 0, 255))
                         else: # uint16
+                            if args['pointcloud'] and "depth_raw" in stream_names and "rectified_right" in stream_names and right_rectified is not None:
+                                if pcl_not_set:
+                                    pcl_converter = PointCloudVisualizer(depthai.get_right_intrinsic(), 1280, 720)
```

Comment:
I also get this error:
AttributeError: module 'depthai' has no attribute 'get_right_intrinsic'

I did a python3 -m pip install -r requirements.txt but maybe a version mismatch?

Reply:
Yes. there is a version mismatch. Need to fix that.

Reply:
Fixed it.
When updating `depthai-python`  `requirements.txt` should be updated too.
See commit : 
6aee211

Reply:
There is issue after update though @saching13 
```
legacy, get_Translation() is not available in version 3
 recalibrate and load the new calibration to the device. 
```

Reply:
Nevermind, I fixed it.